src/blots/Highlight.js

```diff
@@ -0,0 +1,12 @@
+import Inline from 'quill/blots/inline';
+
+import './Highlight.scss';
+
+export default class Highlight extends Inline {
+    static blotName = 'highlight';
+
+    static className = 'pe-highlight';
+
+    // Span is broken, see https://github.com/quilljs/quill/issues/1866
+    static tagName = 'strong';
```

Comment:
what about div with `display: inline`?

Reply:
`strong` appears as more semantic in that case